### ISSUE TITLE
[sass] Canonical way to generate ".min.css" file?

### ISSUE THREAD

**Issue by KelSolaar** [getnikola/plugins#330]
Hi,

I'm trying the _sass_ plugin with bootstrap Scss files and I would like to what it the canonical way to generate the _bootstrap.min.css_ file if any.

In my `targets` file, I can put `bootstrap.scss` and it generates correctly `assets/css/bootstrap.css`, however, I cannot use `bootstrap.min.css`:

```bash
(colour-science.org-USMIstgQ-py3.7) Kali:colour-science.org kelsolaar$ nikola build
Scanning posts..........done!
ERROR: Two different tasks can't have a common target.'output/assets/css/bootstrap.min.css' is a target for copy_assets:output/assets/css/bootstrap.min.css and build_sass:output/assets/css/bootstrap.min.css.
```

Cheers,

Thomas

**Comment by ralsina** [comment]
No canonical way, usually the upstream provides it.

Just use whatever minimizer you like :-)

**Comment by KelSolaar** [comment]
All good and thanks Roberto! I was just checking in case I was missing something obvious!

**Comment by Kwpolska** [comment]
[Just `sass` isn’t enough to build Bootstrap, by the way.](https://chriswarrick.com/blog/2019/02/15/modern-web-development-where-you-need-500-packages-to-build-bootstrap/) I built my Bootstrap pipeline outside of Nikola, you can steal [the Node script for the second step](https://chriswarrick.com/blog/2019/02/15/modern-web-development-where-you-need-500-packages-to-build-bootstrap/#attempt-3-node-clis-are-unnecessary) if you want.

**Comment by KelSolaar** [comment]
To generate the css files, I only needed sass and the scss directory.

**Comment by Kwpolska** [comment]
Yes, but those files are not fully compatible with all browsers. The missing step is running autoprefixer.

**Comment by KelSolaar** [comment]
Good to know, given everything seeming LGTM on Chrome, Firefox, Safari and IE, I don't think I will bother introducing an extra step.
